css/index.css

```diff
@@ -0,0 +1,66 @@
+
+body {
+    background-color: rgb(142, 142, 158);
+    color: white;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    margin: 0;
+    padding-left: 50px;
+    padding-top: 100px;
+    padding-bottom: 50px;
+    padding-right: 60px;
+    /* text-align: center; */
+    /* display: flex; */
+  }
+  .wrapper { 
+    margin: auto;
+    width: 1000px;
+  }
+.img-coding { 
+width: 400px;
+height: 200px;
+}
+
+h1 { 
+    font-size: 20px;
+    color: green;
```

Comment:
You'll also want to change the `font-weight` of the heading to meet the assignment requirements. The requirements also mentions changing these items for other headings too. I think it's alright to fix next week since you clearly know how to do this.